### PR TITLE
LPS-76315

### PR DESCRIPTION
**Pull request by jeyvison**
cc: @pcwhite 

ci:test

### CODE REVIEW

**Comment by jeyvison** [comment]
ci:test

File: portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/portlet/action/RevertWorkflowDefinitionMVCActionCommand.java [hunk 1]
```diff
@@ -110,24 +109,26 @@ protected void doProcessAction(
...
-		addSuccessMessage(actionRequest, workflowDefinition);
+		addSuccessMessage(actionRequest, actionResponse);
 
-		setRedirectAttribute(actionRequest, newWorkflowDefinition);
+		setRedirectAttribute(actionRequest, workflowDefinition);
 
 		sendRedirect(actionRequest, actionResponse);
 	}
 
+	private WorkflowDefinition _previousDefinitionRevision;
```
Comment (marcellustavares) [inline portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/portlet/action/RevertWorkflowDefinitionMVCActionCommand.java]: @jeyvison, this seems odd to me 😕 why are we keeping the `_previousDefinitionRevision` as a class field at this point? keep in mind the action command is a singleton shared across multiple requests.

Comment (jeyvison) [reply portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/portlet/action/RevertWorkflowDefinitionMVCActionCommand.java]: I wasn't counting it as a Singleton but as a Thread-safe component. As it is a singleton you are right about this problem. 

I'll change the implementation to address this

**Comment by jeyvison** [comment]
ci:retest

**Comment by inacionery** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by inacionery** [comment]
Forwarded to https://github.com/liferay/com-liferay-portal-workflow/pull/280

**Pull request closed without merging**